### DISCUSSION TITLE
OpenBSD Dave Gnukem

### DISCUSSION THREAD

**Discussion by davidjoffe** [davidjoffe/dave_gnukem#183] [Announcements]
Just grabbed Matteo Bini's latest Makefile, and had to make just two small changes to get it working on OpenBSD ("gmake -f Makefile.bsd") with the new SDL2 dependencies etc. (though I'm no OpenBSD expert):

![VirtualBox_djOpenBSD_02_12_2022_01_14_56](https://user-images.githubusercontent.com/7451578/205182910-6cbc054d-7115-4bd6-9c25-59e283201e5d.png)

![VirtualBox_djOpenBSD_02_12_2022_01_16_20](https://user-images.githubusercontent.com/7451578/205182930-33583e0d-6b72-4142-8dbd-e657f4efe506.png)

![2022-12-02 01_16_34-Cosmosis - Reality Check - VLC media player](https://user-images.githubusercontent.com/7451578/205182962-29bf5884-44c7-4ea5-b38c-87cc4103a9b6.png)

Added some quick crude basic build documentation here:

https://github.com/davidjoffe/dave_gnukem/tree/main/src/docs/openbsd

Would be great if someone who knows OpenBSD though could maybe improve on it.

**Comment by ibara** [comment]
I sent updated build instructions.
I will update the OpenBSD package.
